packages/cli/src/interfaces.ts

```diff
@@ -110,41 +76,12 @@ export interface IWorkflowToImport
 //            credentials
 // ----------------------------------
 
-export interface ICredentialsBase {
-	createdAt: Date;
-	updatedAt: Date;
-}
-
-export interface ICredentialsDb extends ICredentialsBase, ICredentialsEncrypted {
-	id: string;
-	name: string;
-	shared?: SharedCredentials[];
-}
-
 export type ICredentialsDecryptedDb = ICredentialsBase & ICredentialsDecrypted;
 
 export type ICredentialsDecryptedResponse = ICredentialsDecryptedDb;
 
 export type SaveExecutionDataType = 'all' | 'none';
 
-export interface IExecutionBase {
-	id: string;
-	mode: WorkflowExecuteMode;
-	createdAt: Date; // set by DB
-	startedAt: Date;
-	stoppedAt?: Date; // empty value means execution is still running
-	workflowId: string;
-
-	/**
-	 * @deprecated Use `status` instead
-	 */
-	finished: boolean;
-	retryOf?: string; // If it is a retry, the id of the execution it is a retry of.
-	retrySuccessId?: string; // If it failed and a retry did succeed. The id of the successful retry.
-	status: ExecutionStatus;
-	waitTill?: Date | null;
-}
-
 // Data in regular format with references
 export interface IExecutionDb extends IExecutionBase {
```

Comment:
Are we intentionally leaving behind some `*Db` types like `IExecutionDb` and `ICredentialsDecryptedDb`  here, are we only doing this for types needed in the extensions project currently?

Reply:
Yes! Contrary to their names, `IExecutionDb` and `ICredentialsDecryptedDb` and `IExecutionFlattedDb` are _not_ derived from database entities, either directly or transitively, so they were excluded here. At some point we should rename these ancient types more accurately.